src/scheduler/resv_info.c

```diff
@@ -534,10 +534,10 @@ query_reservations(server_info *sinfo, struct batch_status *resvs)
 			}
 		}
 		resresv_arr[idx] = resresv;
+		resresv_arr[idx + 1] = NULL;
```

Comment:
You're performing an extra add operation by doing it this way. Not a big deal, but I prefer the syntax in your previous comment or...
resresv_arr[idx] = resresv;
resresv_arr[++idx] = NULL;